tests/urls.py

```diff
@@ -3,13 +3,16 @@
 from django.conf.urls import url, include
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
-
 from django.contrib import admin
 admin.autodiscover()
 
+from core import views
+
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^export/category/', views.CategoryExportView.as_view(),
+        name='export-category'),
```

Comment:
maybe use "import_export" for the urls, same as module name.

Reply:
This is not an official `urls.py`, just a test one.